### ISSUE TITLE
Some inline asm fails to parse

### ISSUE THREAD

**Issue by jrmuizel** [NationalSecurityAgency/ghidra#1529]
The following fails to parse with the c parser:
```c
extern __inline__ __attribute__((__always_inline__,__gnu_inline__))
unsigned char _interlockedbittestandset64(long long volatile *Base, long long Offset) { unsigned char old; __asm__ __volatile__ ("lock bts{q %[Offset],%[Base] | %[Base],%[Offset]}" "\n\tsetc %[old]" : [old] "=qm" (old), [Base] "+m" (*Base) : [Offset] "J" "r" (Offset) : "memory" , "cc"); return old; }
```
with:

```
Parse Errors: C Parser:  Encountered errors during parse.
         in /tmp/input.h near line 3
         near token: )
         Last Valid Datatype: _interlockedbittestandset64
         Check around CParserPlugin.out around line: 4
```

**Comment by jrmuizel** [comment]
It looks like this is because of `*Base`. Changing it to `Base` allows parsing to succeed.

**Comment by astrelsky** [comment]
It might actually be the volatile. Try `volatile long long *Base` and see if it takes it. I've seen some funny things where the positioning of unsigned and volatile will make some things parseable/unparseable

**Comment by 0x6d696368** [comment]
Duplicate of #464. Fix in #1082.

**Comment by emteere** [comment]
Parses correctly duplicate of fix as noted above.
